app/controllers/memberships/change_zusatzsektion_to_families_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2025, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+module Memberships
+  class ChangeZusatzsektionToFamiliesController < ApplicationController
+    before_action :authorize, :role, :person, :group
+
+    def create
+      Memberships::ChangeZusatzsektionToFamily.new(role).save!
+
+      redirect_to group_person_path(role.person, group_id: role.group_id), notice: t(".success")
```

Comment:
Im Helper steht der Kommentar "ansonsten wird man bei einer Person ohne Rollen auf die Gruppe der damaligen Mitgliedsrolle redirected. Dadurch werden dann die anderen Personenlinks (Info, Bemerkungen, etc.) mit der falschen group id gebaut und es entsteht ein 404", hier wird nun aber doch auf die Gruppe der Rolle und nicht auf die `group` des :group_id params redirected. Ich sehe gerade nicht ganz, ob ich das nicht verstanden habe oder ob da wirklich etwas nicht aufgeht. Auf jeden Fall wird die `group` in diesem Controller bisher gar nicht verwendet.

Reply:
Merci für den Hinweis. Eine "Person ohne Rollen" darf es in diesem use case gar nicht geben, da die Operation nur auf aktuellen Rollen möglich sein soll. Dies hat in der Spezifikation noch gefehlt, ich habe es ergänzt. 
So erübrigt sich die Diskussion, es kann immer auf die Gruppe des `params[:group_id]` redirected werden.